Author: Paul R. Tagliamonte <paultag@debian.org>
Last-Update: 2014-05-07
Description: Remove code that phones home when doing a version list.
Applied-Upstream: doubt-it
Bug-Debian: http://bugs.debian.org/747140
Forwarded: not-needed

diff --git a/api/client/commands.go b/api/client/commands.go
index 415bdda..f60159f 100644
--- a/api/client/commands.go
+++ b/api/client/commands.go
@@ -386,14 +386,6 @@ func (cli *DockerCli) CmdVersion(args ...string) error {
    }
    fmt.Fprintf(cli.out, "Git commit (server): %s\n", remoteVersion.Get("GitCommit"))
    fmt.Fprintf(cli.out, "Go version (server): %s\n", remoteVersion.Get("GoVersion"))
-   release := utils.GetReleaseVersion()
-   if release != "" {
-       fmt.Fprintf(cli.out, "Last stable version: %s", release)
-       if (dockerversion.VERSION != "" || remoteVersion.Exists("Version")) && (strings.Trim(dockerversion.VERSION, "-dev") != release || strings.Trim(remoteVersion.Get("Version"), "-dev") != release) {
-           fmt.Fprintf(cli.out, ", please update docker")
-       }
-       fmt.Fprintf(cli.out, "\n")
-   }
    return nil
 }
 
